packages/product-components/src/components/SelectAssetModal/SelectAssetModal.tsx

```diff
@@ -167,19 +210,30 @@ export const SelectAssetModal = ({
                     innerAddon={<Icon name="search" variant="tertiary" size="medium" />}
                     innerAddonAlign="left"
                 />
-                <NetworkTabs
-                    networkCategories={networkCategories}
-                    networkCount={networkCount}
-                    searchCategory={searchCategory}
-                    setSearchCategory={setSearchCategory}
-                />
+                {filterCategories?.categoriesType === 'networks' && (
+                    <NetworkTabs
+                        networkCategories={filterCategories.categories}
+                        networkCount={networkCount}
+                        searchCategory={searchCategory}
+                        setSearchCategory={setSearchCategory}
+                    />
+                )}
+                {filterCategories?.categoriesType === 'tokens' &&
+                    setPickedSendTokenCategory &&
+                    pickedSendTokenCategory && (
+                        <SendTokenTabs
+                            sendTokenCategories={filterCategories.categories}
+                            pickedCategory={pickedSendTokenCategory}
+                            setPickedCategory={setPickedSendTokenCategory}
+                        />
+                    )}
 
                 {filteredData.length === 0 ? (
                     <AssetItemNotFound
                         listHeight={LIST_HEIGHT}
                         listMinHeight={LIST_MIN_HEIGHT}
                         searchCategory={searchCategory}
-                        networkCategories={networkCategories}
+                        networkCategories={filterCategories.categories as NetworkFilterCategory[]}
```

Comment:
Perhaps we could find a way to pass only `searchNetworkName` instead of passing `searchCategory` and `networkCategories` and do all login on this component. We will also avoid force typing. What do you think?

Reply:
What do you think about this? https://github.com/trezor/trezor-suite/pull/14799/commits/a7f3d42ace60630dfd61ceca683642ba62c77186

I think it's nice because we'll be able to add a token send form specific not found message if needed.

Reply:
Fine!